charts/kong/values.yaml

```
@@ -581,6 +581,65 @@ postgresql:
     ports:
       postgresql: "5432"
 
+# -----------------------------------------------------------------------------
+# Configure cert-manager integration
+# -----------------------------------------------------------------------------
+
+certificates:
+  enabled: false
+
+  # Set proxy.enabled to true to issue default kong-proxy certificate with cert-manager
+  proxy:
+    enabled: false
+    # Set `issuer` to name of alternate cert-manager clusterIssuer to override default
+    # self-signed issuer. Note: certificate.cluster.enabled must be set to true if this
+    # is not defined with a valid alternative.
+    issuer: ""
+    # Use commonName and dnsNames to set the common name and dns alt names which this
+    # certificate is valid for. Wildcard records are supported by the included self-signed issuer.
+    commonName: "app.example"
+    # Remove the "{}" and uncomment/change the examples to add SANs
+    dnsNames: []
+    # - "app.example"
+    # - "*.apps.example"
+    # - "*.kong.example"
+
+  # Set admin.enabled true to issue kong admin api and manager certificate with cert-manager
+  admin:
+    enabled: false
+    # Set `issuer` to name of alternate cert-manager clusterIssuer to override default
+    # self-signed issuer. Note: certificate.cluster.enabled must be set to true if this
+    # is not defined with a valid alternative.
+    issuer: ""
+    # Use commonName and dnsNames to set the common name and dns alt names which this
+    # certificate is valid for. Wildcard records are supported by the included self-signed issuer.
+    commonName: "kong.example"
+    # Remove the "{}" and uncomment/change the examples to add SANs
+    dnsNames: []
+    # - "manager.kong.example"
+
+  # Set proxy.enabled to true to issue default kong-proxy certificate with cert-manager
```

Is this a copy & paste error?

```suggestion
  # Set portal.enabled to true to issue a kong dev portal certificate with cert-manager
```

---

Thank you. I resolved along with additional cleanup.